### PR TITLE
MathML presentation printing for Range, Min, Max, exp, true, false, None

### PR DESCRIPTION
**Pull request by oscargus**
#### Brief description of what is fixed or changed
Added MathML presentation printing for `Range`, `Min`, `Max`, `exp`, `true`, `false`, `None`

Before:
before161xx

After:
after161xx

#### Release Notes

* printing
    * Added MathML presentation printing for Range, Min, Max, exp, true, false, None

### CODE REVIEW

**Comment by asmeurer** [comment]
Why do you use blackboard font for `e`?

**Comment by asmeurer** [comment]
Also is the last entry in the list a typo? It shows `-oo` instead of `-∞`.

File: sympy/printing/tests/test_mathml.py [hunk 1]
```diff
@@ -1127,6 +1126,36 @@ def test_print_conjugate():
...
+def test_latex_Range():
+    assert mpp.doprint(Range(1, 51)) == \
+        '<mfenced close="}" open="{"><mn>1</mn><mn>2</mn><mi>&#8230;</mi><mn>50</mn></mfenced>'
+    assert mpp.doprint(Range(1, 4)) == '<mfenced close="}" open="{"><mn>1</mn><mn>2</mn><mn>3</mn></mfenced>'
+    assert mpp.doprint(Range(0, 3, 1)) == '<mfenced close="}" open="{"><mn>0</mn><mn>1</mn><mn>2</mn></mfenced>'
+    assert mpp.doprint(Range(0, 30, 1)) == '<mfenced close="}" open="{"><mn>0</mn><mn>1</mn><mi>&#8230;</mi><mn>29</mn></mfenced>'
+    assert mpp.doprint(Range(30, 1, -1)) == '<mfenced close="}" open="{"><mn>30</mn><mn>29</mn><mi>&#8230;</mi><mn>2</mn></mfenced>'
+    assert mpp.doprint(Range(0, oo, 2)) == '<mfenced close="}" open="{"><mn>0</mn><mn>2</mn><mi>&#8230;</mi><mi>&#x221E;</mi></mfenced>'
+    assert mpp.doprint(Range(oo, -2, -2)) == '<mfenced close="}" open="{"><mi>&#x221E;</mi><mi>&#8230;</mi><mn>2</mn><mn>0</mn></mfenced>'
+    assert mpp.doprint(Range(-2, -oo, -1)) == '<mfenced close="}" open="{"><mn>-2</mn><mn>-3</mn><mi>&#8230;</mi><mn>-oo</mn></mfenced>'
```
Comment (asmeurer) [inline sympy/printing/tests/test_mathml.py]: I guess it wasn't a typo. This should be fixed.

**Comment by asmeurer** [comment]
I guess exp is using `exponentiale`. According to https://www.w3.org/TR/MathML2/chapter4.html#contm.exponentiale it renders using a normal `e`, but I guess whatever you made your table with uses a blackboard font? 

Also according to https://www.w3.org/TR/MathML2/chapter4.html#contm.exp there is a `exp` which should be used for the exponential function.

**Comment by asmeurer** [comment]
That is assuming I'm not mixing up presentation and content MathML. I'm not clear how those work in the printer, or which the above document refers to.

**Comment by oscargus** [comment]
Yes, exponential e is used for exp as it was earlier used for printing a single e. Not sure why Firefox and/or the font maps it to blackboard. My rationale was that if there is a special character defined for it, that is the one to be used (rather than discussing if it should be italic or non-italic etc).

I'll check the documentation more carefully, but my guess is that exp is content. Either way we can choose how we want it rendered, but for consistency I think something with e to the power is preferred.

Will get back on this later on.

**Comment by oscargus** [comment]
Yes, the printing of -oo is still in the rather long list of functions to fix. Not sure why it behaves as it does. So the behaviour here is "as expected" given the current code base. (Same thing with complex infinity.)

**Comment by oscargus** [comment]
I've found the issue with -oo and will fix...

 https://github.com/sympy/sympy/blob/34482a7071422e5766a16a0370fa12f9a0b7c8e0/sympy/printing/mathml.py#L295

**Comment by asmeurer** [comment]
I don't know very much about MathML so I'll defer to you on how to best print it. I though we supported both content and presentation MathML, though. 

If the blackboard e is coming from the renderer then we shouldn't worry about it.

**Comment by oscargus** [comment]
Well, there are printers for both Content and Presentation MathML. I've focused on Presentation since it is more "fun" to work with (easy to see the effect of changes etc). Also, Content MathML probably works in more cases already as it much more relies on the actual function names and not on specific printing of it (for example, the range printing was probably OK from a "content" perspective as it was, bad comparison since the content MathML looks different, but the idea hold, printing a function name and arguments is more often OK Content MathML). Finally, some things may not even be relevant/supported from a Content perspective.

So the conclusion is basically that, sure, it would be great if someone worked on the Content MathML as well. Not sure if we shouldn't merge Presentation MathML improvements because of that though.

**Pull request merged**